R/stat-summary.R

```diff
@@ -181,7 +181,11 @@ stat_summary <- function(mapping = NULL, data = NULL,
 StatSummary <- ggproto("StatSummary", Stat,
   required_aes = c("x", "y"),
 
-  extra_params = c("na.rm", "orientation", "fun.data", "fun.max", "fun.min", "fun.args"),
+  default_params = list(
+    na.rm = FALSE, orientation = NA,
+    fun.data = NULL, fun = NULL, fun.max = NULL, fun.min = NULL,
+    fun.args = list()
+  ),
```

Comment:
6. As an example stat, here is how we could implement the default parameters for `StatSummary`